pkg/controller/job/job_controller_test.go

```diff
@@ -1863,7 +1866,7 @@ func TestSyncPastDeadlineJobFinished(t *testing.T) {
 			job := newJobWithName(tc.jobName, 1, 1, 6, batch.NonIndexedCompletion)
 			job.Spec.ActiveDeadlineSeconds = pointer.Int64(1)
 			if tc.setStartTime {
-				start := metav1.NewTime(fakeClock.Now().Add(-time.Second))
+				start := metav1.NewTime(fakeClock.Now())
```

Comment:
+1 for this change, it results in setting the same `startTime` in both test cases, making it easier to follow.